docs/background-services.rst

```diff
@@ -0,0 +1,62 @@
+App background services
+=======================
+
+F5 ACI ServiceCenter has a few background services and tasks running which ensure proper functioning of certain features such as Dynamic Endpoint Discovery. 
+
+View app background services status
+```````````````````````````````````
+1. Click the user menu in the top-right corner.
+
+2. Click **Settings** → **Status**.
+
```

Comment:
2. Click **Settings > Status**    (not sure that the arrow you have there will render, if you know it will, then fine to leave)